usr/share/rear/backup/NETFS/default/41_save_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# save all found capapilities to file 
+
+if [ "$BACKUP_CAP" == "y" ] ; then
+	Log "Saving capabilities to File." 
+	if which getcap >/dev/null 2>&1 ; then
+		getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities
```

Comment:
In my  `$TMP_DIR/backup-include.txt` all my Filesystems are included. So `/`, `/usr` and so on. So I don't see a way to not do an `getcat -r /`.

At least I dont see a better way. Because without `-r` I would have to use find in conjunction with getcap for all includes and eleminate duplicates afterwards.